packages/fast-tooling-react/src/css-editor/editor.tsx

```diff
@@ -68,6 +70,12 @@ export default class CSSEditor extends Foundation<
                     return <CSSWidth value={this.props.data} {...config} />;
                 },
             }),
+            new StandardControlPlugin({
+                id: boxShadowPlugInId,
+                control: (config: ControlConfig): React.ReactNode => {
+                    return <CSSBoxShadow {...config} />;
```

Comment:
can you update the other ones as well? so that there is no confusion for others adding controls.

Reply:
yes, good point. updating now